framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -169,6 +169,20 @@ public void resetRouterTimeout() {
             .endMetadata()
             .done();
     }
+    
+    @Override
+    public void setRouterBalance(String balance) {
+        // Route has a same name as its service.
+        String routeName = getServiceName();
```

Comment:
This is true only for HTTP route, HTTPS route has prefix "secure-" in name. If you want to set Router balance. It will be better to search for routes by label service [in template configured here](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/templates/rhpam74-prod-immutable-kieserver.yaml#L516). By this you should be able to find all routes for the service and then you can set to all of them router balance.

Reply:
Fixed also for setRouterBalance/resetRouterBalance